packages/core/src/dialects/cockroachdb/query-generator-typescript.ts

```diff
@@ -0,0 +1,48 @@
+import { joinSQLFragments } from 'src/utils/join-sql-fragments';
+import type { TableNameOrModel } from '../abstract/query-generator-typescript';
+import { PostgresQueryGenerator } from '../postgres/query-generator';
+
+export class CockroachDbQueryGeneratorTypeScript extends PostgresQueryGenerator {
```

Comment:
You don't need a CockroachDbQueryGeneratorTypeScript file. Instead, write your CockroachDbQueryGenerator file in TypeScript.

We have a bunch of xTypeScript files in the codebase because it's a very old codebase that we're slowly migrating to TypeScript

Reply:
Thanks will do that